README.md

```diff
@@ -203,9 +203,13 @@ The torch wheel version `2.5.0.dev20240820+cpu` can be found at https://download
 
 | Version | GPU Wheel |
 | --- | ----------- |
+| 2.5 (CUDA 12.1 + Python 3.9) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp39-cp39-manylinux_2_28_x86_64.whl` |
 | 2.5 (CUDA 12.1 + Python 3.10) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp310-cp310-manylinux_2_28_x86_64.whl` |
 | 2.5 (CUDA 12.1 + Python 3.11) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp311-cp311-manylinux_2_28_x86_64.whl` |
-| 2.4 (CUDA 12.1 + Python 3.8) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.4.0-cp38-cp38-manylinux_2_28_x86_64.whl` |
```

Comment:
Is this one deprecated?

Reply:
this wheel don't exits, and we don't have trigger for this too: [link](https://github.com/pytorch/xla/blob/6d690904dc92e16a3041e495421f177575a9f4aa/infra/tpu-pytorch-releases/artifacts.auto.tfvars#L118C1-L199C5)